### ISSUE TITLE
Advancements

### ISSUE THREAD

**Issue by slipcor** [AllYourBlocks/ChunkWars#71]
When unlocking the Nether?
When unlocking the End?
When breaking a bed?
When reaching the end of the Puzzle?

**Comment by AllYourBlocks** [comment]
References #25

**Comment by AllYourBlocks** [comment]
Agreed that players should receive the "We Need to Go Deeper" Advancement on first being located in any column in any of the three 16x16 Nether Chunks.

Agreed that players should receive the "The End?" Advancement on first being located in any column in the 48x48 End 'Chunk'.
